x-pack/solutions/chat/plugins/workchat-app/public/application/components/chat/conversation/round_tab_answer.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { ContentRef } from '@kbn/wci-common';
+import { css } from '@emotion/css';
+import {
+  EuiPanel,
+  EuiText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  useEuiTheme,
+  EuiSpacer,
+  EuiFlexGrid,
+  EuiHorizontalRule,
+  EuiIcon,
+  EuiTextColor,
+  useIsWithinBreakpoints,
+} from '@elastic/eui';
+import type { ConversationRound } from '../../../utils/conversation_rounds';
+import { ChatMessageText } from './chat_message_text';
+import { ChatConversationProgression } from './chat_conversation_progression';
+
+interface RoundTabAnswerProps {
+  round: ConversationRound;
+  onSourceClick: (ref: ContentRef) => void;
+}
+
+export const RoundTabAnswer: React.FC<RoundTabAnswerProps> = ({ round }) => {
+  const { euiTheme } = useEuiTheme();
+  const isMobile = useIsWithinBreakpoints(['xs', 's']);
+
+  const { assistantMessage, progressionEvents, loading } = round;
+  const showSources = !loading && (assistantMessage?.citations.length ?? 0) > 0;
+
+  const subTitlesClass = css`
+    font-weight: ${euiTheme.font.weight.bold};
+  `;
+
+  return (
+    <EuiFlexGroup direction="column" gutterSize="none">
+      <EuiFlexItem>
+        <ChatConversationProgression progressionEvents={progressionEvents} />
+        <ChatMessageText
```

Comment:
there is a slight difference in colour from the in-progress streamed text and final response. 


https://github.com/user-attachments/assets/511fa6be-5d5f-48cd-b5ae-21a7122fc632

Reply:
Nice one. I have an opacity animation on the whole block, and it gets triggered when the streaming ends for some reason. Probably a missing key or something.

Reply:
Yeah I figured it out - we redirect to the conversation url after the first message in a new conversation, which causes a rerender of the whole conv, triggering the animation again. New messages don't blink the same way. Not sure how to work around it though